docs/examples.md

```diff
@@ -55,6 +55,20 @@ db.query("INSERT INTO people (name, email) VALUES (?, ?)", name, email);
 ?> Queries like `INSERT INTO` don't return any rows. For these queries `.done()`
 is called automatically.
 
+## Error handling
+
+`DB.query` will throw an exception on failure.
+
+```javascript
+try {
+  db.query("SELEC * FROM people");
```

Comment:
Maybe add a comment, or make the syntax error more obvious
```suggestion
  // Syntax error
  db.query("SELEC * FROM people");
```
or
```suggestion
  db.query("NOT A QUERY");
```

Reply:
Fixed.